src/main/java/io/jenkins/plugins/opentelemetry/job/MonitoringRunListener.java

```diff
@@ -134,6 +138,20 @@ public void _onInitialize(Run run) {
                 .setAttribute(JenkinsOtelSemanticAttributes.CI_PIPELINE_RUN_NUMBER, (long) run.getNumber())
                 .setAttribute(JenkinsOtelSemanticAttributes.CI_PIPELINE_TYPE, OtelUtils.getProjectType(run));
 
+        // CULPRITS
+        Set<User> culpritIds = new HashSet<>(); ;
```

Comment:
Why not default to null

Reply:
Do you mean to avoid sending the attribute if no matches?